src/ui/pages/LockPage/LockPage.tsx

```diff
@@ -145,6 +155,7 @@ const LockPage = () => {
           <h2
             className={`${pageId}-title`}
             data-testid={`${pageId}-title`}
+            id="page-title"
```

Comment:
What is this `id` for? 

And can we replace it with `id={`${pageId}-title`}` so we can make it more specific? (and maybe avoid issues down the line)

Reply:
This id now is not necessary. I will remove it.